src/Components/People.js

```diff
@@ -0,0 +1,73 @@
+import axios from "axios";
+import React from "react";
+
+class People extends React.Component {
+  state = { input: "", people: [] };
+
+  fetchPeople = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people/");
+      debugger;
+      this.setState({ people: res.data });
+      debugger;
+    } catch (error) {
+      this.setState({ people: [] });
+    }
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.fetchPeople();
```

Comment:
It would be better to call fetchPeople in the mount

Reply:
Updated